locales/base/translation.json

```diff
@@ -2140,5 +2140,10 @@
     }
   },
   "pleaseWait": "Please wait",
-  "error": "Error"
+  "error": "Error",
+  "tabBar": {
```

Comment:
naming nit: can we make this more explicit? since we also have a tab bar in the assets page / wallet tab

Reply:
Changed in d428981588b06c5901efb0ed709f27b02cee26b7!